features/networking/metrics.feature

```diff
@@ -0,0 +1,60 @@
+Feature: SDN/OVN metrics related networking scenarios
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-28519
+  @admin
+  Scenario: Prometheus should be able to monitor kubeproxy metrics
+    Given I switch to cluster admin pseudo user
+    And I use the "openshift-sdn" project
+    When I run the :get client command with:
+      | resource | ep            |
+      | n        | openshift-sdn |
+      | o        | json          |
+    And evaluation of `@result[:parsed]['items'][0]['subsets'][0]['addresses'][1]['ip'].to_s` is stored in the :metrics_ep_ip clipboard
+    And evaluation of `@result[:parsed]['items'][0]['subsets'][0]['ports'][0]['port'].to_s` is stored in the :metrics_ep_port clipboard
+    And evaluation of `cb.metrics_ep_ip + ':' +cb.metrics_ep_port` is stored in the :metrics_ep clipboard
+    Given I use the "openshift-monitoring" project
+    And evaluation of `secret(service_account('prometheus-k8s').get_secret_names.find {|s| s.match('token')}).token` is stored in the :sa_token clipboard
+    When I run the :exec admin command with:
```

Comment:
@anuragthehatter is it possible to do the same thing via the external prometheus route?  It so, we can then use the HTTP request method to get the result.

Reply:
Removed redundant token functionality and its clear traffic/query now

Reply:
Re-ran https://privatebin-it-iso.int.open.paas.redhat.com/?c7c9d3aa6479eef4#vfwha8x0Ttf5/oYmaGK99RMX0oqG3nPI7doHpo3QT7Y=